pom.xml

```diff
@@ -137,28 +134,6 @@
             </execution>
           </executions>
         </plugin>
-        <plugin>
```

Comment:
The animal sniffer plugin looks super helpful. Maybe file a bug to use it later for API stability?